### PR TITLE
Update templates with initial feedback

### PR DESCRIPTION
**Pull request by sophies927**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/05-performance.yml [hunk 1]
```diff
@@ -121,48 +121,30 @@ body:
...
     attributes:
       label: Model Type
-      options:
-        - BERT
-        - GPT2
-        - Hugging Face
-        - Longformer
-        - T5
-        - Other / Unknown
+      placeholder: Include your model type and attach the model file (if possible).
```
Comment (faxu) [inline .github/ISSUE_TEMPLATE/05-performance.yml:138]: Looks like there isn't a way to attach files when filing the issue, so suggest to reword this to "Add details about the model architecture and if possible, share the model file"

File: .github/ISSUE_TEMPLATE/06-training.yml [hunk 2]
```diff
@@ -61,39 +73,13 @@ body:
         - ROCm 4.3.1
         - oneDNN
```
Comment (faxu) [inline .github/ISSUE_TEMPLATE/06-training.yml:74]: remove oneDNN, and just keep "CUDA" and "ROCm" - versions can be specified in the EP version field

File: .github/ISSUE_TEMPLATE/07-feature_request.yml [hunk 3]
```diff
@@ -10,24 +9,12 @@ body:
...
+      placeholder: A clear and concise description of the feature you would like, why it should be added, and any alternative solutions or features you've considered. Also provide any additional context necessary for your issue to be addressed (e.g., screenshots, GPU model & memory, etc.).
     validations:
       required: true
   - type: textarea
-    id: alternatives
+    id: use-case
     attributes:
-      label: Alternatives
-      placeholder: A clear and concise description of any alternative solutions or features you've considered.
+      label: Describe scenario use case
```
Comment (faxu) [inline .github/ISSUE_TEMPLATE/07-feature_request.yml:18]: Add placeholder: "The capability and/or customer usage scenario this request supports."

File: .github/labeler.yml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+ep:RockchipNPU:
+  - '(Rockchip NPU)'
+# Add/remove 'ep:TensorRT' label if issue contains the word 'TensorRT'
+ep:TensorRT:
+  - '(TensorRT)'
+# Add/remove 'ep:tvm' label if issue contains the word 'TVM'
+ep:tvm:
+  - '(TVM)'
+# Add/remove 'ep:VitisAI' label if issue contains the word 'Vitis AI'
+ep:VitisAI:
```
Comment (faxu) [inline .github/labeler.yml:32]: Is there a way to add a label for quantization if it's selected in the performance template? (or mentioned anywhere else)

**Comment by faxu** [comment]
One more comment: for the link to Github Discussions, the current text says "questions and discussions to be reviewed by a member of the ORT engineering team"

These can be community-driven discussions as well, right? If so, I'd remove the bit about being reviewed by the ORT team.

File: .github/labeler.yml (hunk 4 above)
Comment (sophies927) [reply .github/labeler.yml:32]: Yes! I will add this to the labeler.yml file so the label is automatically applied.

File: .github/ISSUE_TEMPLATE/05-performance.yml (hunk 1 above)
Comment (sophies927) [reply .github/ISSUE_TEMPLATE/05-performance.yml:135]: There is a way to attach files when filing an issue! Each text box has a note that says "Attach files by dragging & dropping, selecting, or pasting them." Should I still reword?
